evalml/tests/model_understanding_tests/test_graphs.py

```diff
@@ -1028,3 +1032,73 @@ class NotTSPipeline:
     error_msg = "graph_prediction_vs_actual_over_time only supports time series regression pipelines! Received regression."
     with pytest.raises(ValueError, match=error_msg):
         graph_prediction_vs_actual_over_time(NotTSPipeline(), None, None, None)
+
+
+def test_get_tree_data(tree_estimators, logit_estimator):
+    est_class, est_reg = tree_estimators
+    est_logit = logit_estimator
+
+    with pytest.raises(ValueError, match="Tree structure reformatting is not supported for non-Tree estimators"):
+        get_tree_data(est_logit)
```

Comment:
Nit-picky: since we're raising an error for the logistic regressor, we don't really need to actually `fit` it, we can mock it instead (...to save a couple of seconds max ahaha)